### DISCUSSION TITLE
Projeto Mapos foi abandonado?

### DISCUSSION THREAD

**Discussion by rafaelbrasileiro** [RamonSilva20/mapos#2159] [General]
Bom dia.

Gostaria de saber se o projeto Mapos, foi abandonado. Percebo que é bem raro alguém responder sobre Bugs e outros assuntos.

**Comment by fivemdevorg** [comment]
é um projeto Opensource, depende da contribuição dos Devs, eu vou tentar lançar novas atualizações.

**Comment by SuperPorte** [comment]
Muuuuito pelo contrário amigo. Acho que o MapOs nunca esteve mais ativo.
no telegram e whatsapp o MapOs está quente:

https://t.me/joinchat/mbFGDElRFNszNTRh
https://chat.whatsapp.com/D571nPLqNq6JgIsmL6Gucl

Tem uma galera por lá já :smiley:

**Reply by fivemdevorg** [reply]
Eu tenho desenvolvido algumas melhorias para um cliente, podemos falar sobre.
